x-pack/platform/plugins/private/upgrade_assistant/server/lib/es_deprecations_status/migrations.ts

```diff
@@ -153,5 +153,23 @@ export const getEnrichedDeprecations = async (
         ...enrichedDeprecation,
         correctiveAction,
       };
+    })
+    .filter((deprecation, index, elements) => {
```

Comment:
I do start to wonder a little about the performance of this code if there are 1000s of deprecations... but maybe it's premature.